exasol/nb_connector/slc/script_language_container.py

```diff
@@ -65,14 +90,23 @@ def create(
         secrets: Secrets,
         name: str,
         flavor: str,
+        compression_strategy: CompressionStrategy = CompressionStrategy.GZIP,
     ) -> ScriptLanguageContainer:
+        _verify_name(name)
         slc_flavor = SlcFlavor(name)
         if slc_flavor.exists(secrets):
             raise SlcError(
                 "Secure Configuration Storage already contains a"
                 f" flavor for SLC name {name}."
             )
+        slc_compression_strategy = SlcCompressionStrategy(slc_name=name)
```

Comment:
Haha - maybe we should add `SlcSession` again, containing flavor and compression strategy.